Sources/Player/Player.swift

```diff
@@ -186,25 +186,25 @@ public extension Player {
 }
 
 public extension Player {
-    /// Return whether the current player item player can be returned to live conditions.
-    /// - Returns: `true` if skipping to live conditions is possible.
-    func canSkipToLive() -> Bool {
-        guard timeRange.isValidAndNotEmpty, itemDuration.isIndefinite, chunkDuration.isValid else { return false }
-        return time < timeRange.end - chunkDuration
+    /// Return whether the current player item player can be returned to default position thanks to the stream type.
+    /// - Returns: `true` if skipping to default position is possible.
+    func canSkipToDefault() -> Bool {
+        switch streamType {
+        case .onDemand:
+            return true
+        case .dvr where chunkDuration.isValid:
+            return time < timeRange.end - chunkDuration
+        default:
+            return false
+        }
     }
 
-    /// Return the current item to live conditions. Does nothing if the current item is not a livestream or does not
-    /// support DVR.
+    /// Return the current item to default position.
     /// - Parameter completion: A completion called when skipping ends. The provided Boolean informs
     ///   whether the skip could finish without being cancelled.
-    func skipToLive(completion: @escaping (Bool) -> Void = { _ in }) {
-        let time = timeRange.end.clamped(to: timeRange)
-        guard time.isValid else {
-            completion(true)
-            return
-        }
-        queuePlayer.seek(to: time) { [weak self] finished in
-            self?.play()
+    func skipToDefault(completion: @escaping (Bool) -> Void = { _ in }) {
+        let time: CMTime = streamType == .dvr ? timeRange.end.clamped(to: timeRange) : .zero
```

Comment:
If we keep this behavior (see above comment) then I think the following should be a bit more readable:

```suggestion
        let time = (streamType == .dvr) ? timeRange.end.clamped(to: timeRange) : .zero
```